pkg/riff/commands/credential_delete_test.go

```diff
@@ -24,6 +24,27 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+func TestCredentialDeleteOptions(t *testing.T) {
```

Comment:
where/how would you test that the validation is properly "plugged" to its related command? I do not see that integration bit being tested here (which is not extremely important after the fact but is quite natural when test-driving the impl)

Reply:
That test would need to be defined for the command and not the options. In essence, the command would define arguments that would fail validation and then validate that the command failed. Many of the command tests were written before we had option validation, I'll add a test to ensure the options validator is wired properly for each implemented command.

Reply:
done